src/IECoreMaya/FromMayaArrayDataConverter.cpp

```diff
@@ -38,11 +38,33 @@
 
 #include "IECore/VectorTypedData.h"
 
+#include "IECore/GeometricTypedData.h"
+
 using namespace IECore;
 
 namespace IECoreMaya
 {
 
+namespace Detail
```

Comment:
Since this is only used internally to this file, can we move it to an anonymous namespace? Should we also add a specialisation so MVectorArray it treated as GeometricData::Vector?